src/index.ts

```diff
@@ -112,6 +111,12 @@ async function createWebdriverIO(opts: ProgramOpts) {
     )
 
     await runProgram('npx', ['wdio', 'config', ...(useYarn ? ['--yarn'] : []), ...(opts.yes ? ['--yes'] : [])])
+    
+    console.log('Adding scripts to package.json')
+    const isUsingTypescript = fs.existsSync('wdio.conf.ts');
```

Comment:
Let's use existing helper functions, e.g.:
```suggestion
    const isUsingTypescript = await exists('wdio.conf.ts');
```

Reply:
Yes I saw that helper, but to be honest I did not understand it's purpose.. as `fs` can already do the job and is already used a bit above (line 42)

Reply:
`fs.exist` as well as `fs.existSync` are deprecated interfaces. `fs.existSync` doesn't even exist anymore in Node.js v14 and up. It is recommended to use `fs.access` which is what the `exist` helper is doing.

> is already used a bit above (line 42)

That needs to be fixed then as well.

Reply:
Only [fs.exists](https://nodejs.org/api/fs.html#fsexistspath-callback) is deprecated, [fs.existsSync](https://nodejs.org/api/fs.html#fsexistssyncpath) remains a valid API even in Node.js v17.

But anyways, `fs.access` is fine too and I didn't know the `exists` helper was using it 😉

I've made the according changes, except for the line 42 where an async would be needed on the `run` method and I'm not sure about the side effect.